<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customer_page</class>
 <widget class="QWidget" name="customer_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>مشتری</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #34495e;</string>
  </property>
  <widget class="QPushButton" name="print_information">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 24px;

	color:#EA2027; background-color:#bdc3c7; border: 2px solid #ff4d4d;  font-size:24px;</string>
   </property>
   <property name="text">
    <string>نمایش اطلاعات</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_lastname">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:18px; color:black;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;/&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_menu_pc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 24px;

	color:#EA2027; background-color:#bdc3c7; border: 2px solid #ff4d4d;  font-size:18px;</string>
   </property>
   <property name="text">
    <string>برگشت</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;کاربر&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btt_account">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>291</width>
     <height>111</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 24px;

	color:#EA2027; background-color:#bdc3c7; border: 2px solid #ff4d4d;  font-size:24px;</string>
   </property>
   <property name="text">
    <string>شارژ و برداشت از حساب </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_index">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
